supplementary_style_guide/style_guidelines/lead-in-sentences.adoc

```diff
@@ -0,0 +1,11 @@
+[[lead-in-sentences]]
+= Lead-in sentences
+
+A lead-in sentenece is the text that directly follows a procedure or prerequiste heading in a task-based module. It is not the module abstract, which describes the goals of the user for the module.
+
+.Example lead-in sentences
+----
+To register your system to Satellite Server, complete the following steps:
+----
+
+Avoid using a lead-in sentence if it does not add value. Use a lead-in sentence to introduce a procedure or prerequisite only in instances where the lead-in sentence adds context and makes it easier for the user to understand the module content. 
```

Comment:
Suggest specifying "Avoid using a lead-in sentence in the `Prerequisites` or `Procedure` sections of a module if it...".

Reply:
@bergerhoffer @joaedwar I as a writer would need some clear guidance about when it does and when it doesn't add value. I am inclined to say that it rarely adds value. So examples about when it does would be helpful. But again, I believe that @yetanothertw (Vlada) had advised that there are reasons to always include a lead-in sentence.

Reply:
@stoobie. I am also struggling to understand when these add value. I don't see a strong reason to support using lead-in sentences. I think sentences like "To do xyz, complete the following steps:" can never add any real value if a module already has an abstract, but my knowledge is limited, so I would love to see more examples of docs that use lead-ins.